client/checkout/stripe-link/index.js

```diff
@@ -51,6 +51,16 @@ const enableStripeLinkPaymentMethod = ( options ) => {
 			  };
 
 		if ( options.complete_shipping() ) {
+			if (
+				shippingAddress.hasOwnProperty( 'name' ) &&
+				'string' === typeof shippingAddress.name &&
+				shippingAddress.name.length
+			) {
+				const shippingNames = shippingAddress.name.split( ' ' );
```

Comment:
Can we try `const shippingNames = shippingAddress.name.split( ' ', 2 );` here? We'll not need to do any joins later in the code.

Reply:
I am not sure this will work. If we have `Firstname1 Firstname2 Lastname`, it will result in an array with `['Firstname1', 'Firstname2']`

Reply:
```javascript
"Firstname1 Firstname2 Lastname".split(/ (.*)/s, 2);
// -> Array [ "Firstname1", "Firstname2 Lastname" ]
```

Reply:
That's the behavior we agreed to follow.